security/q-feeds-connector/src/opnsense/scripts/qfeeds/lib/log.py

```diff
@@ -58,10 +58,15 @@ def _collect_rule_ids(self):
 
     @staticmethod
     def _parse_log_line(line):
-        # quick scan for datetime, interface, direction, source, dest
+        # quick scan for datetime, interface, direction, source, dest, source_port, dest_port
```

Comment:
looks like this file got stuck as it was also in the other PR. you might need to rebase your branch.